engine/crates/parser-sdl/src/rules/mongodb_directive.rs

```diff
@@ -280,18 +278,16 @@ type Model2 @model(connector: "MongoDB", collection: "Model2") @key(fields: fiel
 
         let registry = crate::parse_registry(config).unwrap();
 
-        for (type_name, tpe) in registry.types.iter() {
+        for (_type_name, tpe) in registry.types.iter() {
             match tpe {
                 engine::registry::MetaType::Object(obj) => {
-                    for (field_name, field) in obj.fields.iter() {
-                        let ty = field.ty.base_type_name();
-                        assert!(registry.types.contains_key(ty), "{type_name}.{field_name} -> {ty}");
+                    for (_field_name, field) in obj.fields.iter() {
```

Comment:
Minor but isn't there a `.values()` function: 

```suggestion
                    for field in obj.fields.values() {
```

Reply:
Done, thanks!